mifosng-android/src/main/java/com/mifos/mifosxdroid/activity/login/LoginActivity.kt

```diff
@@ -1,44 +0,0 @@
-/*
- * This project is licensed under the open source MPL V2.
- * See https://github.com/openMF/android-client/blob/master/LICENSE.md
- */
-package com.mifos.mifosxdroid.activity.login
-
-import android.content.Intent
-import android.os.Bundle
-import androidx.activity.compose.setContent
-import com.mifos.feature.auth.login.LoginScreen
-import com.mifos.mifosxdroid.activity.home.HomeActivity
-import com.mifos.mifosxdroid.activity.setting.UpdateServerConfigFragment
-import com.mifos.mifosxdroid.core.MifosBaseActivity
-import dagger.hilt.android.AndroidEntryPoint
-
-/**
- * Created by ishankhanna on 08/02/14.
- */
-@AndroidEntryPoint
-class LoginActivity : MifosBaseActivity() {
-
-    public override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-
-        setContent {
-            LoginScreen(
-                homeIntent = {
-                    startActivity(Intent(this, HomeActivity::class.java))
-                    finish()
-                }, passcodeIntent = {
-                    startActivity(Intent(this, HomeActivity::class.java))
-                    finish()
-                },
-                onClickToUpdateServerConfig = {
-                    // TODO:: show fragment using navController
-                    UpdateServerConfigFragment().show(
-                        supportFragmentManager,
-                        "UpdateServerConfigFragment"
-                    )
-                }
-            )
-        }
-    }
-}
```

Comment:
This shouldn't be deleted here; it falls under the auth module.

Correct me if I'm wrong, but did you also delete the SettingsActivity.kt file?